htdocs/master.inc.php

```diff
@@ -227,6 +227,7 @@
 
 	// For some countries, we need to invert our address with customer address
 	if ($mysoc->country_code == 'DE' && ! isset($conf->global->MAIN_INVERT_SENDER_RECIPIENT)) $conf->global->MAIN_INVERT_SENDER_RECIPIENT=1;
+	if ($mysoc->country_code == 'DE' && ! isset($conf->global->EXPENSEREPORT_INVERT_SENDER_RECIPIENT)) $conf->global->EXPENSEREPORT_INVERT_SENDER_RECIPIENT=1;
```

Comment:
Why introducing an option just for expense report ? Why not using MAIN_INVERT_SENDER_RECIPIENT ?
This option was introduced for germany. If you need to invert recipient and send on some documents, you probably need to invert it everywhere, doesn't it ?

Reply:
You are absolutely right. I'll change it !